Changelog.md

```diff
@@ -3,7 +3,7 @@
 ## 0.0.1
 
 - completion to create a Database view in *.ddl files
-- added support for ConfigMap / Secret (for both environment variables and datasource properties)
+- added support for Converting to ConfigMap / Secret (for both environment variables and datasource properties)
```

Comment:
Might change "added support for Converting to ConfigMap / Secret (for both environment variables and datasource properties)" to "added support for Converting environment variables and datasource properties to ConfigMap / Secret references"

Reply:
done